lib/ansible/modules/storage/netapp/na_elementsw_account.py

```diff
@@ -86,16 +86,35 @@
     username: "{{ elementsw_username }}"
     password: "{{ elementsw_password }}"
     state: present
+    status: locked
     element_username: TenantA
-    new_element_username: TenantA-Renamed
+
+- name: Rename Account
```

Comment:
is there a difference between renaming and modifying? (line 101). I am not seeing any diff arguments

Reply:
Yes.  There is a from_name for a rename.  In 101, I was asking for an action that would require a rename (from_name) and modify (status: locked) at the same time.

Reply:
roger dodger. thanks for pointing out what my eyes missed @lonico